### DISCUSSION TITLE
Why I cannot open custom theme settings for a single site?

### DISCUSSION THREAD

**Discussion by aranapatona** [darkreader/darkreader#10082] [Q&A]
firefox or chrome latest darkreader, I am not able to open the editor in single site mode,
no matter if I have selected for this site only or for all sites, after clicking dev settings, it always only opens the editor in "all sites mode"
![image](https://user-images.githubusercontent.com/19786351/195422012-b66c5a5e-cd9a-40a6-9ac9-13f7c2954ebd.png)
![image](https://user-images.githubusercontent.com/19786351/195422081-13661d04-c971-4acf-924a-41bfdecc37c7.png)
![image](https://user-images.githubusercontent.com/19786351/195422167-06351ac4-c760-4881-9da9-5e8db6559022.png)

**Comment by bershanskiy** [comment]
The editor is intentionally fairly simplistic because it is not intended for general use, just for development of fixes prior to checking them into a repository and filing PRs on GitHub. In the past I was considering adding an option to "filter" fixes based on a pattern. Basically like a "search" box, which would accept a URL or a fragment and would hide all rules which do not match it. Would that work for you? If not, could you describe a solution you would like to see?

**Comment by jdeamicis** [comment]
On a similar note, how difficult would it be to implement an include file hook into the filter files so that users could edit and store their own personal include (even with the same formatting as this one) for private websites? 

For such websites users would never be allowed to file a PR with the fixes to the filter configs, and having a hook to an external include would prevent the browser extension to ever overwrite any customization in case the add-on is updated.
